templates/common/_base/units/machine-config-daemon-host.service

```diff
@@ -16,7 +16,7 @@ contents: |
   # Need oneshot to delay kubelet
   Type=oneshot
   # TODO add --from-etc-pullspec after ratcheting
-  ExecStart=/usr/libexec/machine-config-daemon pivot
+  ExecStart=/usr/local/bin/machine-config-daemon pivot
```

Comment:
I think we would need to create symlink to usr/local/bin/machine-config-daemon here as well to cover cluster upgrade scenario.